packages/pipeline-editor/src/properties.json

```diff
@@ -47,48 +53,72 @@
         "parameter_ref": "runtime_image",
         "control": "oneofselect",
         "label": {
-          "default": "Runtime Image (docker image used as execution environment)"
+          "default": "Runtime Image"
+        },
+        "description": {
+          "default": "Docker image used as execution environment"
         }
       },
       {
         "parameter_ref": "dependencies",
         "label": {
           "default": "File Dependencies"
         },
-        "place_holder_text": {
```

Comment:
In general, the long texts were to help self-document what users can expect/do with these fields. Are we removing/trimming them due to lack of space?

Reply:
I moved this text into the description because I changed the UI and there is no longer placeholder text for dependencies. I kept the descriptions the same in general, but in this case, I removed two lines because "One filename or expression (e.g. *.py) per line. Supported patterns: ? and *." is unnecessary - now you add entries by pressing the "Add dependencies" button. (see below)
![image](https://user-images.githubusercontent.com/6673460/104340601-79dd5580-54be-11eb-99ab-9a47541585fc.png)

Reply:
With this, are we not allowing the user to just type (*.py)? What happens if the user tries to load a pipeline that already has (*.py) on it? I believe we should still enable users to type (*.py) or (*.csv) as there might be a lot of files? What do others think?

Reply:
I agree. It should still be possible to type in an expression, like it was possible in previous releases.

Reply:
Oh ok - how do you think that should be structured? I can definitely make the fields editable. Should I add a separate button to create a blank field? I could add it next to the "Add dependencies" button, or it could be in the file browser dialog?

Reply:
Something like this? (Add dependency creates a blank entry, Browse files opens the filebrowser) 
![image](https://user-images.githubusercontent.com/6673460/104357397-bfa41900-54d2-11eb-9ef7-d4625975ef10.png)

Reply:
This seems to one of the cases where a special kind of widget would work best - one that allows for the selection of a file or specification of an "arbitrary" string. In essence an amalgamation of what we use today select the input file and to type in output file specifications.